exosphere-shared/src/docker-helper.ls

```diff
@@ -9,6 +9,12 @@ class DockerHelper
     child_process.exec-sync('docker ps -a --format {{.Names}}', 'utf8') |> (.includes container)
 
 
+  @ensure-container-is-running = (container, image) ->
+    return if child_process.exec-sync('docker ps --format {{.Names}}/{{.Status}}' , 'utf8') |> (.includes "#{container}/Up")
+    @remove-container that if @container-exists container
```

Comment:
This is an amazing piece of syntax sugar that I wasn't aware of! It can really help reduce redundancy in a lot of places. Let's apply it more often!